hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -558,44 +579,105 @@ private void initializeSystemManagers(OzoneConfiguration conf,
    */
   private void initializeCAnSecurityProtocol(OzoneConfiguration conf,
```

Comment:
I feel the logic inside this initializeCAnSecurityProtocol can be refactored. Let's discuss offline...